### PR TITLE
source-algolia contribution from btkcodedev

### PR DESCRIPTION
**Pull request by btkcodedev**
Closes https://github.com/airbytehq/airbyte-internal-issues/issues/9840
Auto-generated PR by the Connector Builder for source-algolia

### CODE REVIEW

File: airbyte-integrations/connectors/source-algolia/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+metadataSpecVersion: "1.0"
+data:
+  allowedHosts:
+    hosts:
+      - "*-1.algolianet.com"
```
Comment (natikgadzhi) [inline airbyte-integrations/connectors/source-algolia/metadata.yaml:5]: I think the last line captures the three above

**Review by natikgadzhi — APPROVED** [review]

**Comment by btkcodedev** [comment]
Done

**Pull request merged**